generator-scrivito/generators/obj/index.js

```diff
@@ -0,0 +1,37 @@
+const Generator = require("yeoman-generator");
+const lodash = require("lodash");
+
+module.exports = class extends Generator {
+  start() {
+    this.prompt({
+      type: "input",
+      name: "objClassName",
+      message: "Enter the name of the Obj (e.g. MyPage):",
+    }).then(answers => {
+      const objClassName = answers.objClassName;
+      const folder = `src/Objs/${objClassName}`;
+      const humanFriendlyName = lodash.startCase(objClassName);
+      this._generateFile(
+        "XComponent.js.ejs",
+        `${folder}/${objClassName}Component.js`,
+        { objClassName }
+      );
+      this._generateFile(
+        "XEditingConfig.js.ejs",
+        `${folder}/${objClassName}EditingConfig.js`,
+        { objClassName, humanFriendlyName }
+      );
+      this._generateFile("XClass.js.ejs", `${folder}/${objClassName}Class.js`, {
```

Comment:
That was one renaming to much. It should be `XObjClass.js.ejs` resulting in `${folder}/${objClassName}ObjClass.js`:

```js
      this._generateFile(
        "XObjClass.js.ejs",
        `${folder}/${objClassName}ObjClass.js`,
        { objClassName }
      );
```

Can you rename the template file accordingly?